Document how to change a username (#554)
ocf/ocfweb: ocfweb/docs/docs/staff/procedures/accounts/renaming.md

[[!meta title="Rename an account"]]

> We should sparingly rename an user account, and only do so when it is

Reviewer: ```suggestion
We should sparingly rename a user account, and only do so when it is
```
Interestingly, in English, there is a rule that if a `u` making a `y` sound, you use `a` not `an`. Wtf English.

necessary (e.g. typo in the username).

> To rename a individual account, `sorry` the account with the old name and

Reviewer: ```suggestion
To rename an individual account, `sorry` the account with the old name and
```

> make a `note` of why you are changing the username. Then manually create an

Reviewer: I think it would be good to be explicit about how to do this (IOW, please give an example command line).

account using `approve`, and [[associate|doc staff/procedures/accounts/association]]
the user's `calnetUID` manually. Be sure to `note` the reason and previous

> username in the new account note file.

Reviewer 2: Mention that the `callinkOID` attribute should be removed

#### Note

> When using the OCF web interface the user will see old account as a dropdown option,

Reviewer: ```suggestion
When using the OCF web interface the user will see the old account as a drop-down option,
```

> however user cannot use the old accoun as it is sorried.

Reviewer: ```suggestion
however the user cannot use the old account, as it is sorried.
```

> however user cannot use the old accoun as it is sorried.

Reviewer 2: This is confusing without context. I assume you're referring to the login page here. I don't think it's necessary to mention this.

Author: Yeah, the purpose for this paragraph was to show the implication to the staff who is looking at this document. I can try to clarify the wording.

[[!meta title="Rename an account"]]

We should sparingly rename a user account, and only do so when it is
necessary (e.g. typo in the username).

To rename an individual account, `sorry` the account with the old name and
make a `note` of why you are changing the username. To run `sorry` you need
to be a root user. Follow the prompts of the script.

```

> sorry username_to_rename

Reviewer 3: ```suggestion
sudo sorry username_to_rename
```

This draws the distinction between running `sorry` as root and `note` as your user.
